examples/integrations/kibana/popmon_to_elastic.py

```diff
@@ -0,0 +1,92 @@
+import pickle
+from pathlib import Path
+
+import pandas as pd
+from elastic_connector import ElasticConnector
+
+import popmon  # noqa
+from popmon import resources
+
+if __name__ == "__main__":
+    # Start the Elasticsearch connector
+    es_obj = ElasticConnector()
+
+    # Create an index to store the data
+    es_obj.create_index("popmon-integration")
+
+    # Path to cache the popmon report
+    cache_file = Path("example-report.pkl")
+
+    # Compute if not cached
+    if not cache_file.exists():
+        # open synthetic data
+        df = pd.read_csv(
+            resources.data("flight_delays.csv.gz"), index_col=0, parse_dates=["DATE"]
+        )
+
+        # generate stability report using automatic binning of all encountered features
+        # (importing popmon automatically adds this functionality to a dataframe)
+        report = df.pm_stability_report(
```

Comment:
The stability report produces the results _and_ the report. In this use case, the report is redundant and should be omitted.